src/SilkeyJwtGenerator.js

```diff
@@ -0,0 +1,49 @@
+import ethersjs from "ethers";
+import jwt from "jsonwebtoken";
+
+const {ethers} = ethersjs;
+
+let wallet,
```

Comment:
what's the reason behind using the global scope for `wallet` and `payload`? To my understanding, these two should be provided externally to `signPayload` function

Reply:
Next release